### PR TITLE
docs: update existing articles and screenshots to match version 4.3

### PR DESCRIPTION
**Pull request by thomasheartman**
This updates all images affected by the recent UI changes (excluding the changes from #1185).

I've also updated/added alt text for all the images in the documents I've worked in and have tried to update the text content to reflect recent changes as well.

### CODE REVIEW

**Comment by thomasheartman** [comment]
[Document new toggle view](https://trello.com/c/LUTgWZpn/605-document-new-toggle-view)

File: website/docs/advanced/audit-log.md [hunk 1]
```diff
@@ -3,16 +3,16 @@ id: audit_log
 title: The audit log
 ---
 
-When something is not working as expected it is important to be able to track what changed when, and who performed the change.
+The audit log lets you track changes in Unleash. It lists _what_ changed _when_ it changed, and _who_ performed the change.
```
Comment (chriswk) [inline website/docs/advanced/audit-log.md:6]:
```suggestion
The audit log lets you track changes in Unleash. It lists _what_ changed, _when_ it changed, and _who_ performed the change.
```

File: website/docs/advanced/custom-activation-strategy.md [hunk 2]
```diff
@@ -13,15 +13,15 @@ In this example we want to define an activation strategy offers a scheduled rele
 
 First we need to "define" our new strategy. To add a new "Strategy", open the Strategies tab from the sidebar.
 
-![timestamp_create_strategy](/img/timestamp_create_strategy.png)
+![A strategy creation form. It has fields labeled \"strategy name\" and \"description\". It also has fields for a parameter named \"enableAfter\". The parameter is of type \"string\" and the description is \"Expected format: YYYY-MM-DD HH:MM\". The parameter is required.](/img/timestamp_create_strategy.png)
```
Comment (chriswk) [inline website/docs/advanced/custom-activation-strategy.md:16]: Love the alt texts. Great job!

File: website/docs/user_guide/projects.md [hunk 3]
```diff
@@ -43,48 +43,46 @@ The configuration of a new Project is now available. the following input is avai
 
 ## Deleting an existing project {#deleting-an-existing-project}
 
-To keep your feature toggles clean, removing deprecated projects is important. From the overview of Projects – 
+To keep your feature toggles clean, removing deprecated projects is important. From the overview of Projects –
 1. In the top right of the project card, find the project menu represented by three vertical dots.
 
 
-![Project concept](/img/projects_menu_button.png)
+![A list of projects. Each project has three vertical dots — a kebab-menu — next to it.](/img/projects_menu_button.png)
```
Comment (chriswk) [inline website/docs/user_guide/projects.md:50]: TIL - kebab-menu

I've heard hamburger for vertical, hadn't heard kebab :)

**Review by chriswk — APPROVED** [review]
One missing comma, otherwise LGTM :+1:

File: website/docs/advanced/audit-log.md (hunk 1 above)
Comment (chriswk) [reply website/docs/advanced/audit-log.md:6]: Also, kudos on the correct use of Oxford comma :)

File: website/docs/user_guide/projects.md (hunk 3 above)
Comment (thomasheartman) [reply website/docs/user_guide/projects.md:50]: There's also meatballs, bento, and döner. At least 😋 

image

File: website/docs/advanced/audit-log.md (hunk 1 above)
Comment (thomasheartman) [reply website/docs/advanced/audit-log.md]: Thanks! 😄 And thanks for noticing!

File: website/docs/advanced/api_access.md [hunk 4]
```diff
@@ -11,7 +11,9 @@ Please refer to [Create token](../user_guide/api-token) on how to create an API
 
 Please note that it may take up to 60 seconds for the new key to propagate to all Unleash instances due to eager caching.
 
-> If you need an API token to use in a client SDK you should create a "client token" as these have less access.
+:::note
+If you need an API token to use in a client SDK you should create a "client token" as these have fewer access rights.
+:::
```
Comment (ivarconr) [inline website/docs/advanced/api_access.md:16]: I was not aware of the `:::note` syntax, nice!

Comment (thomasheartman) [reply website/docs/advanced/api_access.md:16]: Yeah, docusaurus has some really nice features built in! 

For admonitions, you also have tip, info, caution, and danger: https://docusaurus.io/docs/markdown-features/admonitions

**Comment by ivarconr** [comment]
Nice job. I really like that you added the alt-texts to images 👍🏼 

One question, should we think about sizing of some of the images? It seems like they will always grow to "100%" width now, and that is not always optimal.

image

**Review by ivarconr — APPROVED** [review]

**Comment by thomasheartman** [comment]
Thanks! I'm no expert, but I try and follow whatever guidelines I can find, both for alt text - such as [this article on good alt texts by big hack](https://bighack.org/how-to-write-better-alt-text-descriptions-for-accessibility/) - and for accessibility in general. Heard about [Holly Tuke's '5 most annoying website features I face as a blind person every single day'](https://bighack.org/5-most-annoying-website-features-i-face-as-a-blind-screen-reader-user-accessibility/) the other day too.

About the image sizing: do you mean that they'll stretch or just that you wouldn't always want them to be 100% wide even if they can be? I couldn't find any clear mention of this in the docusaurus docs, so I can try it out, I suppose.

**Comment by thomasheartman** [comment]
@ivarconr If you were indeed afraid of images stretching past their size (and pixelating), it seems that docusaurus will not do that by default. At least it didn't when I tried it out:

image

**Pull request merged**